packages/api/kanad-test.js

```diff
@@ -0,0 +1,5 @@
+const SDK = require('@api/petstore/index');
```

Comment:
ESLint doesn't like this `require` pattern, but I've tried a bunch of different combinations that don't work 😕

Reply:
The overall ESLint config for our TS code is taking over here and wanting it to be an `import`. You can ignore ESLint here.